src/components/BookItem.css

```diff
@@ -0,0 +1,166 @@
+.book-item {
+  display: flex;
+  justify-content: space-between;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.1);
+  background-color: white;
+  border: solid 1px #f0f0f0;
+  margin: 1rem 6.25rem;
+  padding: 1rem 1.5rem;
+}

```

Comment:
> - Hello @brighton0102, I kindly request you to make style adjustments to the book item and the add book section so that they match the design provided for the navbar. This attention to detail will greatly enhance the overall presentation.👍
> 
> Provided design:
> 
> ![adjust](https://github.com/brighton0102/book-store/assets/40009354/6c4db31d-ca53-4f49-907d-cf17b13bc637)
> 
> Your design:
> 
> ![adjust1](https://github.com/brighton0102/book-store/assets/40009354/f3f6ec2a-eed3-4a84-9b8b-aa6b0a990029)
>